e2e/support/helpers/e2e-embedding-iframe-sdk-helpers.ts

```diff
@@ -217,6 +219,49 @@ export function prepareSdkIframeEmbedTest({
   }
 }
 
+/**
+ * Prepares the testing environment for sdk iframe embedding tests in guest embed mode.
+ */
+export function prepareGuestEmbedSdkIframeEmbedTest({
+  instance = "ee",
+  withTokenFeatures = true,
+  onPrepare,
+}: {
+  instance?: "oss" | "ee";
```

Comment:
nit: I wonder if we could query the runner's env for this like `Cypress.env("MB_EDITION")` instead just to not forget to change the `instance`. but maybe setting it explicitly is better.